VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/wr-participant-list-shared.component.spec.ts

```diff
@@ -254,4 +255,121 @@ describe('WaitingRoom ParticipantList Base', () => {
 
         expect(component.canInvite).toBe(true);
     });
-});
+
+    describe('JudgeParticipantStatusListComponentWithInterpreter', () => {
```

Comment:
Can you move tests for judge participant status lists in the correct place? But by the looks of the test, the description isn't correct

Reply:
This describe looks like a copy and paste of the previous describe. What new scenarios are you actually adding?

Reply:
This looks to be testing functions in the base class `WRParticipantStatusListDirective` and nothing to do functions specific to `JudgeParticipantStatusListComponent`

Reply:
I am checking that the Intrepretee precedes the Interpreter in the participant list and that the refactored UI elements display the correct text based on the Hearing role

Reply:
lets just rename the test here and below :)

Reply:
done